website/src/components/canvas/Canvas.jsx

```diff
@@ -47,13 +63,33 @@ function Canvas() {
       <Sidebar />
       <Box style={styles.box} col={9} sm={12} md={9}>
         <Paper style={styles.paper} zDepth={1}>
-          <h3 style={styles.modelName}>Racecar Model</h3>
+        <TextField style={styles.modelName} id="text-field-default" defaultValue="Model Racecar"/>
           <span style={styles.optionBtnGroup}>
             <FlatButton label="File Options" style={styles.optionBtn} />
             <FlatButton label="View" style={styles.optionBtn} />
             <FlatButton label="Print" style={styles.optionBtn} />
             <FlatButton label="Share" style={styles.optionBtn} />
             <FlatButton label="Resources" style={styles.optionBtn} />
+            <IconButton tooltip="Shaggy" touch={true} tooltipPosition="bottom-center">
```

Comment:
The tooltip icons should be generated in a loop. Use maps to do this (see my sidebar commit for help).